gum/backend-libunwind/gumunwbacktracer.c

```diff
@@ -103,6 +103,9 @@ gum_unw_backtracer_generate (GumBacktracer * backtracer,
 
 #pragma GCC diagnostic push
 #pragma GCC diagnostic ignored "-Wunused-value"
+#if defined(UNW_TARGET_AARCH64) && !defined(__ILP64__)
+#pragma GCC diagnostic ignored "-Wpointer-to-int-cast"
```

Comment:
Is this to silence warnings for code in a header file, or can we fix casts on our end?

Reply:
Removed. I'll work around this with environment variables in the build environment if necessary.